File: app/src/main/java/com/marceljurtz/lifecounter/views/Dicing/IDicingView.java
package com.marceljurtz.lifecounter.views.Dicing;

import com.marceljurtz.lifecounter.contracts.base.IView;
import com.marceljurtz.lifecounter.models.Color;

public interface IDicingView extends IView {
    void setDicingText(String text);
    void loadActivity(Class c);
    void setBackgroundColor(Color color);
}

File: app/src/main/java/com/marceljurtz/lifecounter/views/Dicing/IDicingView.kt
package com.marceljurtz.lifecounter.views.Dicing

import com.marceljurtz.lifecounter.contracts.base.IView
import com.marceljurtz.lifecounter.models.Color

interface IDicingView : IView {
    fun setDicingText(text: String)
    override fun loadActivity(c: Class<*>)
    fun setBackgroundColor(color: Color)
}
